src/router.js

```diff
@@ -37,11 +37,13 @@ const routes = [
 		path: '/_/print/:collective',
 		component: CollectivePrintView,
 		props: (route) => route.params,
+		children: [{ path: ':page*' }],
```

Comment:
Can you give me a quick hint what you want to achieve here? Maybe I oversaw something, but I don't see where this would be used :thinking: